libproject/src/libproject.cpp

```diff
@@ -69,10 +66,10 @@ namespace Interface {
         /*virtual*/ bool open();
 
         /*!
-         * \brief Gives to user list<string> of filenames
-         * \return list<string> of filenames of project or drops exception if project wasn't loaded
+         * \brief Gives to user StrContainer of filenames
+         * \return StrContainer of filenames of project or drops exception if project wasn't loaded
          */
-        /*virtual*/ const list<string> getFileNames() const;
+        /*virtual*/ const StrContainer getFileNames() const;
```

Comment:
#bug
bad name vector<string> is StrContainer by definition
Files, Subprojects are good names for example
it makes possible to substitute std::string for class File and class Subproject for example, which can act like string but have additional behavior